src/Mod/Measure/Gui/TaskMeasure.cpp

```diff
@@ -409,6 +429,25 @@ void TaskMeasure::onSelectionChanged(const Gui::SelectionChanges& msg)
         return;
     }
 
+    // If the control modifier is pressed, the object is just added to the current measurement
+    // If the control modifier is not pressed, a new measurement will be started. If autosave is on,
+    // The old measurement will be saved otherwise discharded
+    if (!(QGuiApplication::keyboardModifiers() & Qt::ControlModifier)) {
```

Comment:
TaskMeasure sets the selection style to greedy, which means that it acts like the Ctrl key is pressed. You might consider to change that.

Reply:
I implemented now the possibility to switch between